addons/medical_ai/functions/fnc_healSelf.sqf

```diff
@@ -0,0 +1,55 @@
+/*
+ * Author: BaerMitUmlaut
+ * Makes the unit heal itself.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+// Player will have to do this manually of course
+if (isPlayer _this) exitWith {};
+// Can't heal self when unconscious
+if (_this getVariable ["ACE_isUnconscious", false]) exitWith {};
+// Check if we're still treating
+if ((_this getVariable [QGVAR(treatmentOverAt), CBA_missionTime]) > CBA_missionTime) exitWith {};
+
+private _needsBandaging = ([_this] call EFUNC(medical,getBloodLoss)) > 0;
+private _needsMorphine  = (_this getVariable [QEGVAR(medical,pain), 0]) > 0.2;
```

Comment:
These two conditions are the same in multiple places. Should we convert those to a macro?